test/api_client/api_resource.js

```diff
@@ -133,14 +136,11 @@ describe('API resource module', () => {
 			it('should reject with error message from server if message is supplied', () => {
 				const serverErrorMessage = 'validation error';
 				const statusCode = 300;
-				popsicleStub.returns({
-					use: () =>
-						Promise.resolve({
-							status: statusCode,
-							body: {
-								message: serverErrorMessage,
-							},
-						}),
+				requestStub.rejects({
```

Comment:
Does axios really treat 300s as errors? Seems strange.

Reply:
it is configurable, but yes
```
  validateStatus: function (status) {
    return status >= 200 && status < 300; // default
  },
````